src/main.ts

```diff
@@ -79,18 +79,18 @@ async function getChangedFiles(token: string, base: string, ref: string, initial
   // This is the simplest case as we don't need to fetch more commits or evaluate current/before refs
   if (base === git.HEAD) {
     if (ref) {
-      core.warning(`'ref' input parameter is ignored when 'base' is set to HEAD`)
+      core.notice(`'ref' input parameter is ignored when 'base' is set to HEAD`)
     }
     return await git.getChangesOnHead()
   }
 
   const prEvents = ['pull_request', 'pull_request_review', 'pull_request_review_comment', 'pull_request_target']
   if (prEvents.includes(github.context.eventName)) {
     if (ref) {
-      core.warning(`'ref' input parameter is ignored when 'base' is set to HEAD`)
+      core.notice(`'ref' input parameter is ignored when 'base' is set to HEAD`)
     }
```

Comment:
This can be deleted too, I think. I don't see any `if` clause here that evaluates `base === git.HEAD` like for the `return` statement above.

https://github.com/dorny/paths-filter/blob/de90cc6fb38fc0963ad72b210f1f284cd68cea36/src/main.ts#L80-L85